frontend/src/pages/Inbox/components/Email.js

```diff
@@ -156,7 +156,15 @@ function EmailAttachments({ props }) {
   );
 }
 
-function EmailInfo({ email }) {
+function EmailInfo({ email, threads, activeThread }) {
+  const activeEmailThread = threads.filter(
+    thread => thread.id === activeThread
+  );
+  const dateReceived = activeEmailThread[0].threadProperties.dateReceived;
+  const date = dateReceived
+    ? moment(dateReceived).format('dddd D MMM YYYY')
+    : format(Date.now(), 'dddd D MMM YYYY');
```

Comment:
I think we should remove `date-fns/format` and just use one library. I think moment is better so just do
`moment().format(dddd D MMM YYYY)` here. Then remove the import above

Reply:
Yeah. Makes sense. Was already there so I didn't know if I should do it or not. Fixed.